ch_tools/chadmin/cli/data_store_group.py

```diff
@@ -15,6 +15,7 @@
 CLICKHOUSE_STORE_PATH = CLICKHOUSE_PATH + "/store"
 CLICKHOUSE_DATA_PATH = CLICKHOUSE_PATH + "/data"
 CLICKHOUSE_METADATA_PATH = CLICKHOUSE_PATH + "/metadata"
+S3_METADATA_STORE_PATH = CLICKHOUSE_PATH + "/disks/object_storage/store"
```

Comment:
What if the disk name Is not `object_storage`? I think we should generate this variable in run-time like:
` CLICKHOUSE_PATH + f"/disks/{disk_name}/store"`

Reply:
It is just a refactoring. We use object_storage [here](https://github.com/yandex/ch-tools/pull/182/files/127dcbfcd73cf8d5890c1be50ac1d7d2744404dc#diff-12dd2989ce7a369b8d062c461ed0d7b0cdb43bb7057f031526cebc6c6f689bebL191)

Let's discuss this in a different PR if you don't mind?